Split speed redux and general movement (#19)
GameBrains/er-core: modules/main-rules/pages/assault-moves-and-charges.adoc

When you move a detachment in the Assault phase, move each of its units one after the other.
For each unit:

> * Unless your unit is subject to a 'may not move' restriction then you *must* move it at least 5 cm -- or into base contact with an enemy unit that is within 5 cm.

Reviewer: ```suggestion
* Where possible, you *must* move it at least 5 cm -- or into base contact with an enemy unit that is within 5 cm.
```

Any movement beyond this is optional.

> * This move *must* be as directly as possible towards an enemy unit.

Reviewer: Hopefully you'll agree that these tweaks resolve the 'may not move versus must move' issue reasonably cleanly.

Author: Where is "a 'may not move' restriction" defined?

Reviewer: I'd argue that we don't need to explicitly define it as it should be reasonably intuitive and the single quote marks hint at this. I did also pre-empt this a bit with some recent tweaks that you may recall, where I switched some other phrases to 'may not move' in the dangerous terrain test and perhaps one or two other places. We can tighten up others as we go along.

Author: For what it's worth, as a player/reader the phrasing made me wonder where I had missed seeing the definition.

Also the tense is confusing, because a unit that gets stopped by a Dangerous terrain test _becomes_ subject to a 'may not move' condition, rather than _already_ being subject to it before moving.

Obviously it's your call as technical editor, but as a reader it's a sentence that requires a lot of processing, and is ambiguous regarding rules that take effect after the unit starts moving.

Reviewer: Hmm. I don't think it's as bad as that. But I can see that there might be something in what you say. Although, I rather suspect that alternatives may be more convoluted.

I suppose the easiest way out is to use a phrase more along the lines of 'If the unit is able to move', like your suggestion in that other context. I didn't use that because I was trying to be a bit more precise and to mirror the 'may not move' phrasing we've started to add in the actual rules that are relevant for this. But perhaps in this case a more vague appeal for the reader to interpret 'able to move' is actually sufficient?

>and is ambiguous regarding rules that take effect after the unit starts moving

Is it though? I mean, aside from common sense, the 'interrupt' rules themselves should make sense. We're really talking about dangerous terrain tests, and you may recall that we tightened up the phrasing there recently for that exact reason.   Those rules explain what happens with a 'mid-move interrupt' — we shouldn't need to repeat it here.

I'll stew on it some more though; and feel free to suggest ideas. We could insert a NOTE (or a footnote, depending on what seems to be the best fit) if we really have to, but it'd be better to weave it directly into the main text if we can.

Author: Thinking aloud, some alternatives would be:

You *must*, where possible, move your unit at least 5 cm...
Subject to movement restrictions, you *must* move your unit at least 5 cm...

Reviewer: Yeah, let's keep it simple and go with: `Where possible, you *must* move it at least`...

GitHub won't let me suggest and apply from this specific thread, because "Applying suggestions on deleted lines is not supported". So, I'll resolve this conversation and apply the change on a new one.

> * This move *must* be as directly as possible towards an enemy unit.

Reviewer: Ugh, slight grammar fix required:
```suggestion
* This move *must* be as direct as possible towards an enemy unit.
```
